manifests/plugins/kafka.pp

```diff
@@ -1,11 +1,15 @@
-class collectd::plugins::kafka ( $connections ) {
-  validate_hash($connections)
+class collectd::plugins::kafka (
+  $modules,
+) {
+  validate_hash($modules)
   Exec { path => [ '/bin/', '/sbin/', '/usr/bin/', '/usr/sbin/' ] }
   include collectd
   include collectd::plugins::java
 
-  collectd::plugins::plugin_common { 'kafka':
-    plugin_file_name     => '20-kafka.conf',
-    plugin_template_name => 'kafka/20-kafka.conf.erb',
+  collectd::plugin { 'kafka':
+    package_name     => 'collectd-kafka',
```

Comment:
This package doesn't exists and should be removed.  RHEL platforms break on this.